.clang-tidy

```diff
@@ -1,2 +1,2 @@
-Checks: '-*,kokkos-*,modernize-use-using'
-FormatStyle: file
+Checks: '-*,kokkos-*,modernize-use-nullptr,kokkos-*,modernize-use-using'
```

Comment:
This doesn't look good. I'll fix that quickly.

Reply:
thx

Reply:
Fixed.